virttest/env_process.py

```diff
@@ -630,6 +631,14 @@ def preprocess(test, params, env):
     if params.get("storage_type") == "nfs":
         image_nfs = nfs.Nfs(params)
         image_nfs.setup()
+        distro_details = distro.detect()
+        if distro_details.name.upper() != 'UBUNTU':
+            selinux_dict = {}
+            selinux_dict['set_sebool_local'] = "yes"
+            selinux_dict['local_boolean_varible'] = "virt_use_nfs"
+            selinux_dict['local_boolean_value'] = "on"
+            seLinuxBool = utils_misc.SELinuxBoolean(selinux_dict)
+            seLinuxBool.setup()
```

Comment:
@will-Do , ubuntu uses apparmor instead of selinux, so I skipped it for ubuntu (`if distro_details.name.upper() != 'UBUNTU'`), for all other distros this NFS setup will be done

Sure, I will params instead of new dict.